spec/index.html

```diff
@@ -732,12 +743,15 @@ <h2>Literals</h2>
         not defined by this specification.</li>
     </ol>
 
-    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>: Two literals are term-equal (the same
-      RDF literal) if and only if the two <a>lexical forms</a>,
-      the two <a>datatype IRIs</a>, and the two
-      <a>language tags</a> (if any) compare equal,
-      character by character. Thus, two literals can have the same value
-      without being the same RDF term. For example:</p>
+    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>:
+      Two literals are term-equal (the same <a>RDF literal</a>)
+      if and only if the two <a>lexical forms</a>,
+      the two <a>datatype IRIs</a>,
+      the two <a>language tags</a> (if any), and
+      the two <a>base directions</a> (if any)
+      compare equal, character by character.
```

Comment:
This is redundant for base direction, but it's easier to apply the comparison criteria for both language tags and base directions.

```suggestion
      compare equal, as if normalized to lower case.
```